projects/packages/videopress/src/client/admin/components/video-storage-meter/index.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { Text } from '@automattic/jetpack-components';
+import { __, sprintf } from '@wordpress/i18n';
+import classnames from 'classnames';
+import filesize from 'filesize';
+/**
+ * Internal dependencies
+ */
+import ProgressBar from '../progress-bar';
+import styles from './style.module.scss';
+import { VideoStorageMeterProps } from './types';
+import type React from 'react';
+
+/**
+ * Video Storage Meter component
+ *
+ * @param {VideoStorageMeterProps} props - Component props.
+ * @returns {React.ReactNode} - VideoStorageMeter react component.
+ */
+const VideoStorageMeter: React.FC< VideoStorageMeterProps > = ( { className, total, used } ) => {
+	if ( ! total || used == null ) {
+		return null;
+	}
+
+	const progress = used / total;
+	const progressLabel = `${ ( progress * 100 ).toFixed() }%`;
+	const totalLabel = filesize( total, { base: 2 } );
+
+	return (
+		<div className={ classnames( className ) }>
+			<Text className={ classnames( styles.text ) }>
```

Comment:
kind of ambiguous name :-p Please ignore it

Reply:
You mean the `.text`? Yeah. How about something like `.label` or `.percentage`?

Reply:
`percentage-description`?